src/servers/Server_Zone/DebugCommand/DebugCommandHandler.cpp

```diff
@@ -594,10 +594,10 @@ void Core::DebugCommandHandler::script( char* data, Entity::Player &player, boos
       if( subCommand == params )
          player.sendDebug( "Command failed: requires name of script to reload" );
       else
-         if( g_scriptMgr.getNativeScriptHandler().reloadScript( params ) )
-            player.sendDebug( "Reloaded '" + params + "' successfully" );
-         else
-            player.sendDebug( "Failed to reload '" + params + "'" );
+      {
+         g_scriptMgr.getNativeScriptHandler().reloadScript( params );
+         player.sendDebug( "Attempting to reload script: " + params );
```

Comment:
Is there any feedback of whether this actually succeeded? 
reload should probably return a bool to indicate that.

Reply:
reloading happens on the next server tick to avoid a rare condition where the lock on `/cache/<script>.dll` still exists and can't be overwritten so you can't really return from it anymore with useful information

it does show up in the server log if it is successful or not though. was debating on whether it would be a good idea to sendDebug to anyone logged in with a gmlvl > 0 that reloads were successful or not but I was thinking the server log would suffice

Reply:
So it is not a reload call but rather a queueing of a reload process, this should probably be indicated by the name to reduce ambiguity.

Reply:
sounds good, will change that